src/Preferences/AppearancePreferencesViewController.m

```diff
@@ -55,12 +64,6 @@ - (void)viewWillAppear {
     // Do view setup here.
     [self initializePreferences];
     
-    // Set up to be notified if preferences change outside this window
```

Comment:
I think the `NSNotificationCenter` code should stay in `viewWillAppear` because we don't need to set them up unless the user is actually going to visit those particular preference panes. This would be the same for all `*PreferencesViewController.m` files.